tests/cypress/tests/standalone/select_row_column_test.ts

```diff
@@ -0,0 +1,79 @@
+import DashTable, { DashTableHelper } from 'cypress/DashTable';
+
+import { BasicModes, AppMode, AppFlavor } from 'demo/AppMode';
+
+Object.values(BasicModes).forEach(mode => {
+    describe(`select row, mode=${mode}`, () => {
+        beforeEach(() => {
+            cy.visit(`http://localhost:8080?mode=${mode}`);
+            DashTable.toggleScroll(false);
+        });
+
+        describe('fe pagination & sort', () => {
+            it('can select row', () => {
+                DashTable.getSelect(0).within(() => cy.get('input').click());
+                DashTable.getSelect(0).within(() => cy.get('input').should('be.checked'));
+            });
+
+            it('can select row when sorted', () => {
+                cy.get('tr th.column-0 .column-header--sort').last().click({ force: true });
+                DashTable.getSelect(0).within(() => cy.get('input').click());
+                DashTable.getSelect(0).within(() => cy.get('input').should('be.checked'));
+            });
+
+            it('select, sort, new row is not selected', () => {
+                DashTable.getSelect(0).within(() => cy.get('input').click());
+                DashTable.getSelect(0).within(() => cy.get('input').should('be.checked'));
+                cy.get('tr th.column-0 .column-header--sort').last().click({ force: true }).click({ force: true });
+                DashTable.getSelect(0).within(() => cy.get('input').should('not.be.checked'));
+            });
+        });
+    });
+});
+
+describe('select rows & columns in multiple tables without id', () => {
```

Comment:
Renamed the file to take into consideration that there are now rows and columns tests.

Create two tables without ids and check that row and column selections are not shared between them.